client/ayon_max/plugins/load/load_redshift_proxy.py

```diff
@@ -71,6 +65,22 @@ def switch(self, container, context):
         self.update(container, context)
 
     def remove(self, container):
-        from pymxs import runtime as rt
         node = rt.GetNodeByName(container["instance_node"])
         remove_container_data(node)
+
+    def _create_redshift_object_type(self):
```

Comment:
Should this be `_get_redshift_object_type()` or even just `_get_object_type()`? Since the function itself doesn't necessarily "create" anything?
```suggestion
    def _create_redshift_object_type(self):
```

Reply:
i would prefer get